odoolint/tests/test_convert_file.py

```diff
@@ -178,6 +181,7 @@ def test_70_ref_unreachable_csv(self):
         self.create_unreachable('base.group_user')
         self.create_imd('security/ir.model.access.csv', 'data', 2, 1)
 
+    @unittest.skipIf(ODOO_PATCHED is True, "Odoo is patched with str2bool")
```

Comment:
Got it!
I think I have to do a PR dummy pointing to my branch in the following lines in `maintainer-quality-tools`:
https://github.com/Vauxoo/maintainer-quality-tools/blob/master/travis/travis_install_nightly#L117-L125